src/main.ts

```diff
@@ -141,41 +142,44 @@ export default class LatexSuitePlugin extends Plugin {
 
 		this.showSnippetsLoadedNotice(snippets.length, Object.keys(snippetVariables).length,  becauseFileLocationUpdated, becauseFileUpdated);
 
-		return sortSnippets(snippets);
+		return snippets;
```

Comment:
I believe it's necessary to keep the call to `sortSnippets()` here. When loading snippets from files, sorting needs to be done on the complete set of snippets from all snippet files — i.e. after all of the files have been loaded. `parseSnippets()` only sorts snippets _within_ files.